### ISSUE TITLE
update from version 10.0.1 to 9.1.1.5 ?

### ISSUE THREAD

**Issue by stanelie** [nextcloud/server#1581]
### Steps to reproduce
1. get notification of available 10.0.1 upgrade
![2016-09-29 11_36_40-hote de l experience windows shell](https://cloud.githubusercontent.com/assets/2935029/18961477/34e21408-863b-11e6-8980-040a9323e4e0.png)
2. open upgrader
3. 

### Expected behaviour
The upgrader should propose an upgrade to the next version

### Actual behaviour
The upgrader proposes a downgrade
![2016-09-29 11_36_57-hote de l experience windows shell](https://cloud.githubusercontent.com/assets/2935029/18961478/34e73b5e-863b-11e6-949e-155c9034a787.png)

### Server configuration
**Operating system**:
 Ubuntu 14.04.5 LTS
**Web server:**
Apache
**Database:**
MySQL
**PHP version:**
PHP 5.5.9-1ubuntu4.19
**Nextcloud version:** (see Nextcloud admin page)
10.0.0
**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated
**Where did you install Nextcloud from:**
Nextcloud website
**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**The content of config/config.php:**
From config.php:  ` 'updater.server.url' => 'https://updates.nextcloud.org/updater_server/',

**Are you using external storage, if yes which one:** local/smb/sftp/...
Yes : local, SMB
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
ActiveDirectory

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>
```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by LukasReschke** [comment]
> From config.php: ` 'updater.server.url' => 'https://updates.nextcloud.org/updater_server/',

Don't configure such settings in config.php unless you really know what you are doing. In this case, this updater server is only compatible with the new updater shipped with 10.0.1. Please remove that line and perform a manual update.

Also 9.1.1.5 > 9.1.0.16.

**Comment by stanelie** [comment]
Hello.

Well, 9.1.1.5 (the updater proposed version) < 10.0.0 (my current installed version), so, it would qualify as a downgrade.

That line in config.php was given to me some time ago, when I first migrated from owncloud to Nextcloud, I believe. Maybe the documentation should be updated to reflect this? I had no idea it should be removed after I was done migrating.

**Comment by LukasReschke** [comment]
> Well, 9.1.1.5 (the updater proposed version) < 10.0.0 (my current installed version), so, it would qualify as a downgrade.

9.1 is the internal version code for 10.0. The new updater shipped in 10.0.1 is able to display that properly.

**Comment by stanelie** [comment]
Hello.

Where can I find a table with this correspondence information? I was not aware that 9.1 == 10.0.

So, to upgrade from 10.0.0 to 10.0.1, I have to do it manually?
